packages/framework/collabspaces/src/collabSpaces.ts

```diff
@@ -537,7 +587,7 @@ export class CollabSpacesRuntime
 	private getCellInfo(rowArg: number, colArg: number): ICellInfo {
 		const row = rowArg + 1;
 		const col = colArg + 1;
-		assert(row > 0 && col > 0, "arguments");
+		assert(row >= 0 && col >= 0, "arguments");
```

Comment:
I'd not make this change - see comments above. All you need in updateReverseMap() is getting the value from matrix - call it matrix directly. This function is doing a lot of work that is not needed for your case, and it's better for it to be specialized to work with "normal" cells, not mapping cells

Reply:
good point.